lms/templates/footer.html

```diff
@@ -38,7 +42,7 @@
             ## can be included in the footer on other sites
             ## (e.g. a blog or marketing front-end) to provide a consistent
             ## user experience.  See the branding app for details.
-            <img alt="organization logo" src="${footer['logo_image']}">
+            <img alt="${_('organization logo')}" src="${footer['logo_image']}">
```

Comment:
Nit: This is a very strange alt tag. I suppose it is because this footer is intended to be customized. I find it a strange thing to have localized, because now the translation teams will now need to think about what it should say. I'd be tempted not to localize this, or if you want to make it more realistic I'd plug in the actual organization name from settings.